test_app/after/src/inline_items.erl

```diff
@@ -277,8 +277,13 @@ long_arglist(X1,
              VeryVeryLongName1,
              VeryVeryLongName2,
              VeryVeryLongName3) ->
-    X1 + X2 + X3 + Y1 + VeryVeryLongName1 + VeryVeryLongName2 +
-        VeryVeryLongName3.
+    X1
```

Comment:
I know you'll most surely reply "because `erl_prettypr`" here, but why? Why does `+` have to be handled the same as `++`?
Does
```
1 + 1
```
become 
```
1
+ 1
```
?

Reply:
Because it is an infix operator. This time is not prettypr's fault. It's `erl_syntax`

Reply:
But
```erlang 
1
+ 1
```
won't be forced by default, will it?

Reply:
Not at all. It will only be indented if _the whole expression_ should be indented.